2025-01-16 10:35:07,311 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2025-01-16 10:35:07,311 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Configure stats pid to 100900
2025-01-16 10:35:07,311 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Loading settings from /user/ma4177/.config/wandb/settings
2025-01-16 10:35:07,311 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Loading settings from /user/ma4177/Exp_neural/wandb/settings
2025-01-16 10:35:07,312 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-01-16 10:35:07,312 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-01-16 10:35:07,312 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main_run.py', 'program_abspath': '/user/ma4177/Exp_neural/main_run.py', 'program': '/user/ma4177/Exp_neural/main_run.py'}
2025-01-16 10:35:07,313 INFO    MainThread:100900 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-01-16 10:35:07,314 INFO    MainThread:100900 [wandb_init.py:_log_setup():521] Logging user logs to /user/ma4177/Exp_neural/wandb/run-20250116_103507-4afs9bfx/logs/debug.log
2025-01-16 10:35:07,315 INFO    MainThread:100900 [wandb_init.py:_log_setup():522] Logging internal logs to /user/ma4177/Exp_neural/wandb/run-20250116_103507-4afs9bfx/logs/debug-internal.log
2025-01-16 10:35:07,316 INFO    MainThread:100900 [wandb_init.py:init():561] calling init triggers
2025-01-16 10:35:07,317 INFO    MainThread:100900 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'epochs': 1000, 'do_dev_every_n_epochs': 1, 'print_results_every_n_epochs': 1, 'save_model': False, 'epochs_between_save': 1, 'choose_best_model_on': 'dev_loss', 'load_previous_model': False, 'load_model_path': 'models/saved_models/your_model.pt', 'save_model_folders': ['2025_01_16', 'hybrid_policy'], 'save_model_filename': 1737041706, 'logging_params': {'use_wandb': True, 'wandb_project_name': 'inventory_control', 'wandb_entity': 'alvomatias', 'exp_name': 'hybrid_general', 'env_name': 'inventory'}, 'problem_params': {'n_stores': 1, 'n_warehouses': 0, 'n_extra_echelons': 0, 'lost_demand': True, 'maximize_profit': False, 'is_hybrid': True, 'discrete_features': {'fixed_ordering_cost': {'thresholds': [-0.0001, 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0], 'values': [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]}}}, 'observation_params': {'include_warehouse_inventory': False, 'include_static_features': {'holding_costs': True, 'underage_costs': True, 'procurement_costs': True, 'lead_times': True, 'upper_bounds': False, 'mean': False, 'std': False}, 'demand': {'past_periods': 0, 'period_shift': 0}, 'include_past_observations': {'arrivals': 0, 'orders': 0}, 'time_features_file': None, 'time_features': None, 'sample_features_file': None, 'sample_features': None}, 'params_by_dataset': {'train': {'n_samples': 8192, 'batch_size': 1024, 'periods': 50, 'ignore_periods': 16}, 'dev': {'n_samples': 8192, 'batch_size': 1024, 'periods': 50, 'ignore_periods': 16}, 'test': {'n_samples': 8192, 'batch_size': 1024, 'periods': 30, 'ignore_periods': 16}}, 'use_wandb': True, 'wandb_project_name': 'inventory_control', 'wandb_entity': 'alvomatias', 'exp_name': 'hybrid_general', 'env_name': 'inventory'}
2025-01-16 10:35:07,318 INFO    MainThread:100900 [wandb_init.py:init():611] starting backend
2025-01-16 10:35:07,319 INFO    MainThread:100900 [wandb_init.py:init():615] setting up manager
2025-01-16 10:35:07,320 INFO    MainThread:100900 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-01-16 10:35:07,322 INFO    MainThread:100900 [wandb_init.py:init():623] backend started and connected
2025-01-16 10:35:07,325 INFO    MainThread:100900 [wandb_init.py:init():715] updated telemetry
2025-01-16 10:35:07,341 INFO    MainThread:100900 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2025-01-16 10:35:07,441 ERROR   MainThread:100900 [wandb_init.py:init():774] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-01-16 10:35:07,482 ERROR   MainThread:100900 [wandb_init.py:init():1199] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/user/ma4177/.conda/envs/exp_neural/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    run = wi.init()
  File "/user/ma4177/.conda/envs/exp_neural/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 780, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-01-16 10:35:08,485 WARNING MsgRouterThr:100900 [router.py:message_loop():77] message_loop has been closed
